demo/index.html

```diff
@@ -159,6 +166,69 @@
           .catch(onError);
         });
 
+        startNotificationsButton.addEventListener('click', function() {
+          heartRateDevice.request().then(function(device) {
+            return heartRateMeasurement.startNotifications().then(function() {
+              template.text = device.name + ' Notifications session has started.';
+              progressBar.indeterminate = false;
+              heartRateMeasurement.addEventListener('value-changed', parseHeartRate);
```

Comment:
I'm not sure to understand what you mean there.
The `value-changed` event is automatically generated by Polymer.

The internal `_valueChanged` observer for the `value` property is already used for the `autowrite` attribute.